src/Components/AlertBox/AlertBox.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './AlertBox.css';
```

Comment:
Do we want to use CSS or something like this? 
https://material-ui.com/styles/basics/